docs/README.md

```diff
@@ -77,6 +77,9 @@ In the example above, the [Rich Text](https://vtex.io/docs/components/all/vtex.r
 | Prop name | Type | Description | Default value |
 | --- | --- | --- | --- |
 | `trigger` | `Enum` | Whether the Modal content should be triggered by user click ( `click`) or when the page is fully loaded (`load`)  | `'click'` |
+| `openPerSection` | `Boolean` | Use a cookie to open the modal just once, when the trigger is `load`                                    |  `false`  |
+| `expiresDateCookie` | `number` | How many days of validity the `openPerSection` cookie should have.                              |    `10`   |
```

Comment:
I don't know if days is the best unit here, what do you guys think? @vtex-apps/store-framework-devs ?

Reply:
And if you're making the change to make `openPerSection` just an option of `trigger`, it should be something saying that this prop only works if `trigger='on-load-per-section'` (not really sure about this name too, so open to suggestions)

Reply:
Yeah, I'd change from `days` to at least hours or even minutes. I don't think anything less than minutes would be required 🤔

Reply:
I like hours

Reply:
@Klynger, @kaisermann I have done the changes that you ask for. Now the modal validity is in hours and the new trigger to load per section!!